x/nft/types/msgs.go

```diff
@@ -77,75 +74,7 @@ func (msg MsgTransferNFT) GetSigners() []sdk.AccAddress {
 
 // MsgEditNFTMetadata edits an NFT's metadata
 type MsgEditNFTMetadata struct {
-	Owner           sdk.AccAddress
-	ID              uint64
-	Denom           string
-	EditName        bool
-	EditDescription bool
-	EditImage       bool
-	EditTokenURI    bool
-	Name            string
-	Description     string
-	Image           string
-	TokenURI        string
-}
-
-// NewMsgEditNFTMetadata is a constructor function for MsgSetName
-func NewMsgEditNFTMetadata(owner sdk.AccAddress, id uint64,
-	editName, editDescription, editImage, editTokenURI bool,
-	denom, name, description, image, tokenURI string,
-) MsgEditNFTMetadata {
-	return MsgEditNFTMetadata{
-		Owner:           owner,
-		ID:              id,
-		Denom:           strings.TrimSpace(denom),
-		EditName:        editName,
-		EditDescription: editDescription,
-		EditImage:       editImage,
-		EditTokenURI:    editTokenURI,
-		Name:            strings.TrimSpace(name),
-		Description:     strings.TrimSpace(description),
-		Image:           strings.TrimSpace(image),
-		TokenURI:        strings.TrimSpace(tokenURI),
-	}
-}
-
-// Route Implements Msg
-func (msg MsgEditNFTMetadata) Route() string { return RouterKey }
-
-// Type Implements Msg
-func (msg MsgEditNFTMetadata) Type() string { return "edit_metadata" }
-
-// ValidateBasic Implements Msg.
-func (msg MsgEditNFTMetadata) ValidateBasic() sdk.Error {
-	if msg.Owner.Empty() {
-		return sdk.ErrInvalidAddress("invalid owner address")
-	}
-	if !msg.EditName && !msg.EditDescription && !msg.EditImage && !msg.EditTokenURI {
-		return ErrEmptyMetadata(DefaultCodespace, "")
-	}
-	return nil
-}
-
-// GetSignBytes Implements Msg.
-func (msg MsgEditNFTMetadata) GetSignBytes() []byte {
-	bz := cdc.MustMarshalJSON(msg)
-	return sdk.MustSortJSON(bz)
-}
-
-// GetSigners Implements Msg.
-func (msg MsgEditNFTMetadata) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{msg.Owner}
-}
-
-/* --------------------------------------------------------------------------- */
-// MsgMintNFT
-/* --------------------------------------------------------------------------- */
-
-// MsgMintNFT defines a MintNFT message
-type MsgMintNFT struct {
-	Sender      sdk.AccAddress
-	Recipient   sdk.AccAddress
```

Comment:
@fedekunze same question about whether we were going to leave these in and just not let them be called unless explicit in the app's implementation.

Reply:
see comment above